source/uwp/Renderer/lib/AdaptiveActionSetRenderer.cpp

```diff
@@ -4,15 +4,20 @@
 
 #include "AdaptiveActionSet.h"
 #include "AdaptiveActionSetRenderer.h"
+#include "AdaptiveShowCardActionRenderer.h"
```

Comment:
Is this needed?

Reply:
nope... looks like some of the other ones are also unneeded. Probably snuck in while I was trying to get things building :)

---
In reply to: [325838868](https://github.com/microsoft/AdaptiveCards/pull/3457#discussion_r325838868) [](ancestors = 325838868)